clarkio/vscode-twitch-highlighter
# Add Font Color Setting

### Purpose
These changes add a new setting that allows users to define the color of the font within a highlight. This comes as a result from the following issue raided by @corbob: https://github.com/clarkio/vscode-twitch-highlighter/issues/53

 - Addresses: https://github.com/clarkio/vscode-twitch-highlighter/issues/53

### Files Changed

- `extension.ts`: add use of `color` key in decoration type setup which is read from settings or defaults to `white`
- `package.json`: defined new setting for font color so that it is available to users
- `README.md`: update documentation to reflect the new setting option and changelog details

### How to Test

- Run in terminal/command line: `git clone https://github.com/clarkio/vscode-twitch-highlighter.git`
- Run in terminal/command line: `cd vscode-twitch-highlighter && npm install`
- Press F5 or click debug button (green triangle/play button)
- Create a highlight to see the result

### What to Validate

- Make sure the visible font color of a highlight is defaulting to white
- Open settings in Extension Host instance of VS Code and make sure there is a setting available called "Highlight Font Color" (search for "Twitch Color").
- After changing the default font color to something other than white (such as "red") create a new highlight and confirm the font color in the highlight is now red
